Sources/PopupBridge/PopupBridgeUserScript.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+struct PopupBridgeUserScript {
+    let scheme: String
+    let scriptMessageHandlerName: String
+    let host: String
+    
+    var rawJavascript: String {
+        return """
+        ;(function () {
```

Comment:
🤔 : I wonder if that leading `;` does anything? Maybe we could remove.

Reply:
Yeh, great question! I'll mess around with this, see if it's necessary. This was just a 1:1 copy of what existed in ObjC.

Will def make a follow-up PR though to simplify that JS snippet if we can.